web/src/services/dashboards.ts

```diff
@@ -27,20 +27,19 @@ const dashboards = {
       `/api/${organization}/dashboards?page_num=${page_num}&page_size=${page_size}&sort_by=${sort_by}&desc=${desc}&name=${name}`
     );
   },
-  create: (organization: string, dashboardID: String, data: any) => {
-    return http().post(`/api/${organization}/dashboards/${dashboardID}`, data, { headers: { 'Content-Type': 'application/json; charset=UTF-8' } });
+  create: (organization: string, data: any) => {
+    return http().post(`/api/${organization}/dashboards`, data, { headers: { 'Content-Type': 'application/json; charset=UTF-8' } });
```

Comment:
NB @ktx-kirtan

Reply:
Yes, will take care of frontend calls. As it will need some other refactors as well.